src/server_manager/web_app/ui_components/app-root.ts

```diff
@@ -372,6 +372,11 @@ export class AppRoot extends polymerElementWithLocalize {
       #getConnectedDialog .buttons {
         margin-top: -5px; /* undo spacing added after iframe */
       }
+      @media (max-width: 887px) {
```

Comment:
It seems Material uses 840dp as the breakpoint: https://m3.material.io/foundations/layout/applying-layout/window-size-classes

Maybe we should follow that?